elyra/kfp/bootstrapper.py

```diff
@@ -557,8 +563,13 @@ def package_list_to_dict(cls, filename: str) -> dict:
                         package_name, package_version = line.strip("\n").split(sep=" @ ")
                     elif "===" in line:
                         package_name, package_version = line.strip("\n").split(sep="===")
-                    else:
+                    elif "==" in line:
                         package_name, package_version = line.strip("\n").split(sep="==")
+                    elif line.startswith("-e "):
```

Comment:
We should support short and long form https://pip.pypa.io/en/stable/cli/pip_install/#install-editable
```suggestion
                    elif line.startswith("-e ") or line.startswith("--editable "):
```